README.md

```diff
@@ -1,127 +1,151 @@
-# Analyze your code for free with SonarCloud
+# Scan your code with SonarQube Cloud [![QA](https://github.com/SonarSource/sonarcloud-github-action/actions/workflows/qa.yml/badge.svg)](https://github.com/SonarSource/sonarcloud-github-action/actions/workflows/qa.yml)
 
-This SonarSource project, available as a GitHub Action, scans your projects with SonarCloud, and helps developers produce 
-[Clean Code](https://www.sonarsource.com/solutions/clean-code/?utm_medium=referral&utm_source=github&utm_campaign=clean-code&utm_content=sonarqube-scan-action).
+> [!WARNING]
+> This action is deprecated and will be removed in a future release. 
+> Please use the `sonarqube-scan-action` action instead. 
+> The `sonarqube-scan-action` is a drop-in replacement for this action.
 
-<img src="./images/SonarCloud-72px.png">
+This SonarSource project, available as a GitHub Action, scans your projects with SonarQube [Cloud](https://www.sonarsource.com/products/sonarcloud/).
 
-[SonarCloud](https://www.sonarsource.com/products/sonarcloud/) is a widely used static analysis solution for continuous code quality and security inspection. 
-It helps developers identify and fix issues in their code that could lead to bugs, vulnerabilities, or decreased development velocity.
-SonarCloud supports the most popular programming languages, including Java, JavaScript, TypeScript, C#, Python, C, C++, and [many more](https://www.sonarsource.com/knowledge/languages/).
+![Logo](./images/SQ_Logo_Cloud_Dark_Backgrounds.png#gh-dark-mode-only)
+![Logo](./images/SQ_Logo_Cloud_Light_Backgrounds.png#gh-light-mode-only)
```

Comment:
Having both dart and light logos with `#gh-dark-mode-only` and `#gh-light-mode-only` fragments make the logo responsive to the theme of the user.